zun_kata.md

```diff
@@ -0,0 +1,148 @@
+# Introduction
+
+This guide describes how to get Kata containers working with OpenStack Zun using DevStack on Ubuntu 16.04.  It is based on the following links at the time of writing this guide.  Running DevStack with this guide will setup Docker and Clear Containers 2.0 which you will replace with Kata Containers.  
```

Comment:
Suggested rewrite: This guide describes how to get Kata containers to work with OpenStack Zun using DevStack on Ubuntu 16.04.  Running DevStack with this guide will setup Docker and Clear Containers 2.0, which you replace with Kata Containers. Currently, the instructions are based on the following links: